storybook/stubs/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -3,6 +3,9 @@ pragma Singleton
 
 import QtQml 2.15
 
+import utils 1.0
+import Models 1.0
```

Comment:
is it necessary?

Reply:
oh I was taking `ModelsData.icons.socks` from there